test/Dapr.E2E.Test.Actors/IPingActor.cs

```diff
@@ -0,0 +1,15 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Dapr.Actors;
+
+namespace Dapr.E2E.Test.Actors
+{
+    public interface IPingActor : IActor
+    {
+        Task Ping();
+    }
```

Comment:
This is the best way I could find that verify that the actors runtime is healthy.